backend/canvas_app_explorer/management/commands/manage_pylti.py

```diff
@@ -0,0 +1,66 @@
+import json
+from Crypto.PublicKey import RSA
+from Crypto.PublicKey.RSA import RsaKey
+from django.core.management.base import BaseCommand, CommandParser
+from django.db.utils import IntegrityError
+
+# https://github.com/dmitry-viskov/pylti1.3/blob/master/pylti1p3/contrib/django/lti1p3_tool_config/models.py
+from pylti1p3.contrib.django.lti1p3_tool_config.models import LtiToolKey, LtiTool
+
+
+class Command(BaseCommand):
+    help = "Generate a private/public key pair and add them to the pylti13 database. This can be called with the same issuer + client ID to update a previous key. The Tool Key will be re-used"
```

Comment:
Please check this help string for accuracy, clarity. Sounds initially like it just generates the key pair, but it does more than that.

Reply:
Thanks, I re-read this and updated it to describe a little more what it's doing.